### PR TITLE
Use utc timezone so storage and helper functions work correctly

### PR DESCRIPTION
**Pull request by duttonw**

### CODE REVIEW

File: ckanext/data_qld/actions.py [hunk 1]
```diff
@@ -174,7 +176,7 @@ def create_datarequest(original_action, context, data_dict):
     data_req = db.DataRequest()
     _undictize_datarequest_basic(data_req, data_dict)
     data_req.user_id = context['auth_user_obj'].id
-    data_req.open_time = datetime.datetime.now()
+    data_req.open_time = datetime.datetime.now(tzinfo=pytz.utc)
```
Comment (ThrawnCA) [inline ckanext/data_qld/actions.py:179]: Why not just use `datetime.utcnow`?

Comment (duttonw) [reply ckanext/data_qld/actions.py]: done

File: ckanext/data_qld/reporting/helpers/helpers.py [hunk 2]
```diff
@@ -45,8 +45,8 @@ def get_report_date_range(request):
     if end_date:
         end_date = toolkit.h.date_str_to_datetime(end_date)
     else:
-        ckan_timezone = toolkit.config.get('ckan.display_timezone', None)
-        end_date = datetime.now(pytz.timezone(ckan_timezone))
+        # interal ckan datetimes is utc, presentation is ckan_timezone
```
Comment (ThrawnCA) [inline ckanext/data_qld/reporting/helpers/helpers.py:48]: "interal" should be "internal"

**Comment by duttonw** [comment]
@ThrawnCA  is this still required?

**Comment by ThrawnCA** [comment]
@duttonw Looks like it's still relevant, yes.

I just reopened and re-closed https://staging.data.qld.gov.au/datarequest/523fd0cd-1309-4a65-97ee-1f334a9b804c and it showed a closing date of "-1 days ago", indicating that the timestamp of 10:22 was being interpreted as UTC time, ie 20:22 locally.

**Pull request merged**